package.json

```diff
@@ -92,6 +96,7 @@
     "react-native-tab-view": "^2.11.0",
     "react-native-vector-icons": "^6.6.0",
     "react-native-web": "^0.11.7",
+    "react-native-webview": "8.1.1",
```

Comment:
@kennethlien @wangannie Did this come with firebase? Are we using it? If not let's remove it